config/routes.rb

```diff
@@ -3,4 +3,9 @@
 
   # Defines the root path route ("/")
   # root "articles#index"
+  resources :amusement_parks, only: [:show]
+
+  resources :mechanics, only: [:show] do
+    post "add_ride", on: :member
+  end
```

Comment:
These routes mostly look good, but line 9 does not adhere to restful conventions. There are a couple different ways you could have done this, but could have done nested routing with `resources :rides, only: [:create]` if you wanted the uri to be `"/mechanics/:id/rides"`. That's one option, but `add_ride` is not one of our restful routes.